### PR TITLE
Avoid overwriting `Base.convert`

### PR DESCRIPTION
**Pull request by sloede**

### CODE REVIEW

**Comment by sloede** [comment]
I will of course not merge before all checks pass, which is contingent on #15

File: src/Trixi2Img.jl
Comment (ranocha) [inline src/Trixi2Img.jl]: That's bad on Windows... You should rename that file to be usable on case-insensitive systems.

File: test/test_trixi2img.jl
Comment (ranocha) [inline test/test_trixi2img.jl]: You also need to update the lower bound on Trixi.jl in `test/Project.toml` to `v0.2.0` for this to work.

**Review by ranocha — CHANGES_REQUESTED** [review]

File: test/test_trixi2img.jl
Comment (sloede) [reply test/test_trixi2img.jl]: Yes, I am waiting here for (hopefully) you to fix #15 ;-)

File: src/Trixi2Img.jl
Comment (sloede) [reply src/Trixi2Img.jl]: Ugh, you're right. Any suggestions? `driver.jl`? `main.jl`?

Comment (sloede) [reply src/Trixi2Img.jl]: Good catch, by the way!

Comment (ranocha) [reply src/Trixi2Img.jl]: `convert.jl`? `driver` or `main` are not really more descriptive, I think. But it's up to you - I don't have a strong opinion here.

File: test/test_trixi2img.jl
Comment (ranocha) [reply test/test_trixi2img.jl]: Done

Comment (sloede) [reply test/test_trixi2img.jl]: Will you also merge #16?

File: src/Trixi2Img.jl
Comment (sloede) [reply src/Trixi2Img.jl]: OK, I'll go back to `convert.jl`

File: test/test_trixi2img.jl
Comment (ranocha) [reply test/test_trixi2img.jl]: I've made a necessary modification there. I will merge it when the tests pass - and you don't beat me with that :wink:

Comment (ranocha) [reply test/test_trixi2img.jl]: Merged. You may need to rebase on master.

**Review by ranocha — APPROVED** [review]
LGTM! Can be merged when the tests pass.

File: test/test_trixi2img.jl
Comment (ranocha) [inline test/test_trixi2img.jl]:
```suggestion
const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples")
```

Comment (ranocha) [reply test/test_trixi2img.jl]: The additional `"2d"` is added via a `joinpath` in https://github.com/trixi-framework/Trixi2Img.jl/blob/75a8e42dc336be7272702502b395b9e3b269dc1d/test/test_2d.jl#L14

Comment (sloede) [reply test/test_trixi2img.jl]: Fixed in 71f521b

**Pull request merged**
